### ISSUE TITLE
Not sure how to use plugin from HS

### ISSUE THREAD

**Issue by jrosebr1** [Py-Image/dpd-helpscout-chrome#1]
Please see my messages in Slack. I have everything installed and running but I don't see any option anywhere to reenable the download.

**Comment by jrosebr1** [comment]
Okay, I found the issue. I needed to create the HS app via:

https://secure.helpscout.net/apps/custom/

However, I was using:

https://secure.helpscout.net/users/apps/

Which appears to be different. Once I used the form rather than the latter it started working.

However, this makes me concerned regarding the Drip Tags custom app:

https://github.com/Py-Image/dpd-helpscout-wordpress/issues/2

As I think once they deprecate the old API the Drip Tags will stop working.
